sbc_compassion/models/import_letters_history.py

```diff
@@ -59,32 +60,33 @@ class ImportLettersHistory(models.Model):
         ("saved", _("Saved"))], compute="_set_ready")
     nber_letters = fields.Integer(_('Number of letters'), readonly=True,
                                   compute="_count_nber_letters")
-    # path where the zipfile are store
-    path = "/tmp/sbc_compassion/"
-    # list zip in the many2many
-    # use ';' in order to separate the files
-    list_zip = fields.Text("LIST_ZIP", readonly=True)
+    is_mandatory_review = fields.Boolean("Mandatory Review", default=False)
 
     data = fields.Many2many('ir.attachment', string=_("Add a file"))
     letters_line_ids = fields.Many2many('import.letter.line')
     letters_ids = fields.Many2many('sponsorship.correspondence')
 
-    @api.one
+    ##########################################################################
+    #                             FIELDS METHODS                             #
+    ##########################################################################
+    @api.multi
     @api.depends("letters_line_ids", "letters_line_ids.status",
                  "letters_ids", "data")
     def _set_ready(self):
-        check = True
-        for i in self.letters_line_ids:
-            if i.status != "ok":
-                check = False
-        if check and len(self.letters_line_ids) > 0:
-            self.state = "ready"
-        elif len(self.letters_ids) > 0:
-            self.state = "saved"
-        elif len(self.letters_line_ids) > 0:
-            self.state = "pending"
-        else:
-            self.state = "draft"
+        for letters in self:
```

Comment:
The api.one will be deprecated in the next version, therefore I am trying to avoid it